### PR TITLE
e2e/framework: security/ and /service remove direct import /k8s.io/kubernetes/pkg/

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**
 
/kind cleanup
 
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

ref:https://github.com/kubernetes/kubernetes/issues/74352

**Special notes for your reviewer**:

e2e/framework: security/ and /service remove direct import /k8s.io/kubernetes/pkg/

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
Thanks for doing this!
/approve

File: test/e2e/framework/security/apparmor.go [hunk 1]
```diff
@@ -37,6 +36,12 @@ const (
 
 	loaderLabelKey   = "name"
 	loaderLabelValue = "e2e-apparmor-loader"
+
+	// Unconfined profile
+	profileNameUnconfined = "unconfined"
+
+	// The prefix to an annotation key specifying a container profile.
+	containerAnnotationKeyPrefix = "container.apparmor.security.beta.kubernetes.io/"
```
Comment (andrewsykim) [inline test/e2e/framework/security/apparmor.go]: Add a TODO comment to import this from the `k8s.io/api` once it's moved there

File: test/e2e/framework/security/apparmor.go [hunk 2]
```diff
@@ -37,6 +36,12 @@ const (
 
 	loaderLabelKey   = "name"
 	loaderLabelValue = "e2e-apparmor-loader"
+
+	// Unconfined profile
+	profileNameUnconfined = "unconfined"
```
Comment (andrewsykim) [inline test/e2e/framework/security/apparmor.go]: same here

**Review by andrewsykim — COMMENTED** [review]
minor comments to add TODO, lgtm otherwise :)

File: test/e2e/framework/security/apparmor.go (hunk 1 above)
Comment (andrewsykim) [reply test/e2e/framework/security/apparmor.go]: ref https://github.com/kubernetes/kubernetes/pull/89225

Comment (tanjunchen) [reply test/e2e/framework/security/apparmor.go]: > Add a TODO comment to import this from the `k8s.io/api` once it's moved there

sure Thanks you review

Comment (tanjunchen) [reply test/e2e/framework/security/apparmor.go]: > ref #89225
updated.
@andrewsykim    I will follow this pr 89225

File: test/e2e/framework/security/apparmor.go (hunk 2 above)
Comment (tanjunchen) [reply test/e2e/framework/security/apparmor.go]: done

**Comment by oomichi** [comment]
The change itself seems good for me, please rebase this for the latest master.

**Comment by tanjunchen** [comment]
> The change itself seems good for me, please rebase this for the latest master.

done

**Pull request merged**